data/json/professions.json

```diff
@@ -3658,22 +3658,18 @@
     "items": {
       "both": {
         "items": [
-          "pants_army",
-          "socks",
-          "undershirt",
+          "nomex_socks",
           "boots_combat",
           "wristwatch",
-          "fancy_sunglasses",
-          "gloves_tactical",
-          "helmet_army",
+          "nomex_gloves",
+          "flight_helmet",
```

Comment:
I think you should add battery to helmet, same for zombie drop
`{ "item": "flight_helmet", "ammo-item": "light_plus_battery_cell", "charges": 150 }`

Reply:
> I think you should add battery to helmet, same for zombie drop
> `{ "item": "flight_helmet", "ammo-item": "light_plus_battery_cell", "charges": 150 }`

I agree. I think they should mostly contain atomic batteries as i think they were becoming the standard with the military at the time of the cataclysm. I will get around to it when I can which may not be this week but other editors are free to correct this if they care enough to.  There is no rush, anyhow.  Thanks @Tairesh :)

Reply:
@Tairesh I have a question regarding this, Does `{ "item": "flight_helmet", "charges": [ 0, 100 ] }` do the same thing you propose with your line? As I already did that with one of them and I think that was the intention.

Reply:
I don't think it would be a good solution to add a random charge level to things in professions. You can just press Enter and adjust the charge level.

https://user-images.githubusercontent.com/1105196/123194346-c4c4b580-d4bf-11eb-8853-063ca5a43f3a.mp4

Reply:
Cool. Did it like you said.

 I wanted to change the entire loadout to be like the 'official US mil loadout I used for the NPC, such as having flares, whistle, flashlight, etc.  But decided that was beyond the scope and put it on my list... Prod me if I forget to do wider profession overhaul of pilot. :)